pebbles/views/blueprints.py

```diff
@@ -131,6 +133,27 @@ def put(self, blueprint_id):
         db.session.commit()
 
 
+class BlueprintCopy(restful.Resource):
+    @auth.login_required
+    @requires_group_manager_or_admin
+    def put(self, blueprint_id):
+        user = g.user
+        blueprint = Blueprint.query.filter_by(id=blueprint_id).first()
+        if not blueprint:
+            abort(404)
+
+        if not user.is_admin and not is_group_manager(user, blueprint.group):
+            logging.warn("invalid group for the user")
```

Comment:
Hmm, If the user is not an admin or a group manager of that blueprint group, then he shall not be allowed the group